client/packages/common/src/utils/formatters/formatters.ts

```diff
@@ -56,4 +56,8 @@ export const Formatter = {
       .trim();
     return _str.substring(0, 1).toUpperCase() + _str.substring(1);
   },
+  isoNoMs: (date?: Date | null): string | null => {
```

Comment:
This naming isn't clear to me what is NoMs?

Reply:
Changed it to naiveDateTime